patches/chrome-elevation_service-service_main.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/elevation_service/service_main.h b/chrome/elevation_service/service_main.h
+index 27f962e0d0a63dd7fa5552609eaab19216d90c5a..70eb383258e90a7c35415a6feacf04cfa1379827 100644
+--- a/chrome/elevation_service/service_main.h
++++ b/chrome/elevation_service/service_main.h
+@@ -107,6 +107,8 @@ class ServiceMain {
+   // service control manager asks the service to exit.
+   base::WaitableEvent exit_signal_;
+ 
++  static const wchar_t* GetWindowsServiceName();
```

Comment:
I think we're really overcomplicating this. These patches could be done in a chromium_src override, but since you're already patching https://github.com/brave/brave-core/pull/20754/files#diff-ca904f946de4232d3d7de33623f0d4a276516c08288579e761b1410ebf50b79cR9 just hard code `BraveElevationService` there

Reply:
or perhaps as discussed in DM there's actually an upstream bug here? Ideally we should submit an upstream patch to avoid all this hard coded duplication in the first place

Reply:
Simplified this in https://github.com/brave/brave-core/pull/20754/commits/13a56a34418bec5670f4cdb2a932818e1e647de5 and then opened up a convo in Chromium slack:
https://chromium.slack.com/archives/CGL0978B1/p1701214590182519

Will report back what I find there. Should be possible to cause a bad repro step by uninstalling regular Chrome and triggering this on Chrome Beta for example. If that's the case, I can create a crbug and submit a patch

Reply:
OK I did some testing - it does seem this string value passed in isn't used - since the service is registered as `SERVICE_WIN32_OWN_PROCESS`. MSDN mentions it can't be null - but you can use `""`. I verified locally that putting a fake service name, installing, then calling elevation service works - it calls the appropriate one 🎉 

More info here:
https://learn.microsoft.com/en-us/windows/win32/api/winsvc/ns-winsvc-service_table_entrya

I added a commit to remove this patch entirely. Thanks for the Chromium team for sharing the notes behind the issue

Reply:
Removed in https://github.com/brave/brave-core/pull/20754/commits/be363eb58f75fe24dc5a5aab72240e123d6cac53